tests/Config/ConfigTest.php

```diff
@@ -1483,4 +1483,70 @@ public function pluginRegistersScannerAndAnalyzer(int $flags, ?int $expectedExce
         self::assertSame(get_class($analyzerMock), $config->getFiletypeAnalyzers()[$extension] ?? null);
         self::assertNull($expectedExceptionCode, 'Expected exception code was not thrown');
     }
+
+    public function testTypeStatsForFileReporting(): void
```

Comment:
tests are failing on windows. I'm pretty sure this has to do with path separators in getcwd()

Reply:
Sorry, I've no idea how to approach this issue. I don't have access to windows machine.
`testBarebonesConfig()` also uses `getcwd()`. I wonder how it is working.

Should I try using `dirname(__DIR__, 2)` instead?

Reply:
oh, that's not getcwd(), the issue is with `. '/'` at the end of paths. You can replace with ` . DIRECTORY_SEPARATOR`

Reply:
Cool, it worked. Thanks.
I should be careful with dir separator from next time.